src/Behat/Gherkin/Loader/CucumberNDJsonAstLoader.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+namespace Behat\Gherkin\Loader;
+
+use Behat\Gherkin\Node\BackgroundNode;
+use Behat\Gherkin\Node\ExampleTableNode;
+use Behat\Gherkin\Node\FeatureNode;
+use Behat\Gherkin\Node\OutlineNode;
+use Behat\Gherkin\Node\ScenarioInterface;
+use Behat\Gherkin\Node\ScenarioNode;
+use Behat\Gherkin\Node\StepNode;
+
+/**
+ * Loads a feature from cucumber's protobuf JSON format
+ */
+class CucumberNDJsonAstLoader implements LoaderInterface
+{
+
+    public function supports($resource)
+    {
+        return is_string($resource);
+    }
+
+    public function load($resource)
+    {
+        return array_map(
```

Comment:
you should either filter out `null` values or forbid `getFeature` from returning it